packages/instrument-utils/src/storedInstrument/ManageAccountInstrumentsTable/__snapshots__/ManageAccountInstrumentsTable.test.tsx.snap

```diff
@@ -0,0 +1,282 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ManageAccountInstrumentsTable matches snapshot with rendered output 1`] = `
```

Comment:
We should not be having this file anymore 🤔

Reply:
we don't use snapshots with RTL?

Reply:
it's a freshly generated snapshot

Reply:
No, we are going to be moving away from snapshots. Since that's not the goal for testing using RTL.

Reply:
For example https://thesametech.com/snapshot-testing-in-rtl/